README.md

```diff
@@ -1,18 +1,16 @@
 # LeaderBoard-Project
```

Comment:
- [ ] In your README documentation, kindly include instructions on how users can set up your app locally.
- [ ] Also include commands on how to install packages (`npm install`) and run the app (`npm start` or `npm run build`). This can be of great help to users